src/unk_0203F6C4.c

```diff
@@ -1633,39 +1633,35 @@ static BOOL ScrCmd_End (ScriptContext * ctx)
     return FALSE;
 }
 
-static BOOL ScrCmd_003 (ScriptContext * ctx)
+static BOOL ScrCmd_WaitFrames (ScriptContext * ctx)
 {
-    FieldSystem * fieldSystem = ctx->fieldSystem;
-    u16 v1 = ScriptContext_ReadHalfWord(ctx);
-    u16 v2 = ScriptContext_ReadHalfWord(ctx);
-    u16 * v3 = sub_0203F118(fieldSystem, v2);
+    FieldSystem *fieldSystem = ctx->fieldSystem;
+    u16 frames = ScriptContext_ReadHalfWord(ctx);
+    u16 countdownVar = ScriptContext_ReadHalfWord(ctx);
+    u16 *countdown = sub_0203F118(fieldSystem, countdownVar);
 
-    *v3 = v1;
+    *countdown = frames;
 
-    ctx->data[0] = v2;
+    ctx->data[0] = countdownVar;
 
-    ScriptContext_Pause(ctx, sub_0203F70C);
-    return 1;
+    ScriptContext_Pause(ctx, ScriptContext_WaitTimer);
+    return TRUE;
 }
 
-static BOOL sub_0203F70C (ScriptContext * ctx)
+static BOOL ScriptContext_WaitTimer (ScriptContext * ctx)
```

Comment:
[suggestion] `ScriptContext_WaitTimer` -> `ScriptContext_DecrementTimer`

`Wait` implies that the routine will loop until a certain state is achieved. `Decrement` more clearly describes the behavior.